src/mpyl/utilities/docker/__init__.py

```diff
@@ -92,3 +92,17 @@ def build(logger: Logger, root_path: str, file_path: str, image_tag: str, target
     stream_docker_logging(logger=logger, generator=logs, task_name=f'Build {file_path}:{target}')
     logger.debug(logs)
     return True
+
+
+def get_env_variables(project: Project, run_properties: RunProperties) -> dict[str, str]:
+    if project.deployment is None:
+        raise KeyError(f'No deployment information was found for project: {project.name}')
+    if len(project.deployment.properties.env) == 0:
+        raise KeyError(f'No properties.env is defined for project: {project.name}')
+
+    env_variables: dict[str, str] = {}
+
+    for env_variable in project.deployment.properties.env:
+        env_variables[env_variable.key] = env_variable.get_value(run_properties.target)
```

Comment:
```suggestion
    env_variables: dict[str, str] = {
        env_variable.key: env_variable.get_value(run_properties.target) for env_variable in
        project.deployment.properties.env
    }
```
is more pythonic I think, though maybe not more readable..

Reply:
Fine by me, although I have to say that for someone without a lot of Python experience the first method was easier